atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -438,6 +443,59 @@ AVMediaType ParseMediaType(const std::string& media_type) {
   }
 }
 
+bool SystemPreferences::IsTouchIDAvailable() {
+  if (@available(macOS 10.12.2, *)) {
+    base::scoped_nsobject<LAContext> context([[LAContext alloc] init]);
+    return [context
+        canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics
+                    error:nil];
```

Comment:
I _think_ this is just returning that _some_ biometric is available, but that it could be face or touch.

https://stackoverflow.com/a/48901055 indicates we might also want to check context's biometryType for LABiometryTypeTouchID

Reply:
I thought about this, but the challenge is that `LABiometryType` is an enum that wasn't added until `10.13.2`. So we could, but _many_ users are still on 10.12 so we'd be locking out a whole upgrade's worth of users by doing that...what do you think?